va/va.c

```diff
@@ -36,8 +36,31 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#if defined(_WIN32)
+#include <windows.h>
+#include <io.h>
+#define strtok_r strtok_s
+#define dlsym GetProcAddress
+#define dlclose FreeLibrary
+#define dlopen(a, b) LoadLibrary(a)
```

Comment:
I can't speak for what Intel's preference will be, but generally, I'd prefer to introduce an abstraction that doesn't exactly match *either* native platform API, like you did with `library_t`, rather than pretending that the platforms support an API set that belongs to a different platform.

Also, instead of using `#define`, prefer `inline` functions.